UefiPayloadPkg/SPI/device.h

```diff
@@ -0,0 +1,91 @@
+#ifndef DEVICE_H
```

Comment:
This file will probably be needed only for reading the PCI config space. Since EDK2 provides the library for reading the PCI config space, this should be obsolete. If you need pci_read_config* function, use MdePkg/Library/UefiPciLibPciRootBridgeIo/PciLib.c

Reply:
Fixed

Reply:
@IgorBagnucki remove the file

Reply:
Removed